less/common.less

```diff
@@ -113,6 +122,11 @@ html.no-js .no-js-warning {
   border-top: 0.2rem solid @white;
 }
 
+#content {
+  padding-left: 0;
+  padding-right: 0;
+}
```

Comment:
arghhh, if `#content` does have left/right padding we won't be able to have a full width section with background colour (e.g., the quote section on the new homepage).  I probably have to go through all the pages and make sure thing are being wrapped within extra inner containers... :sob: